frontend/js/src/utils/APIService.ts

```diff
@@ -1635,4 +1661,27 @@ export default class APIService {
     await this.checkStatus(response);
     return response.json();
   };
+
+  searchPlaylistForUser = async (
```

Comment:
For consistency with the other method below:
```suggestion
  searchPlaylistsForUser = async (
```